commons-statistics-descriptive/src/test/java/org/apache/commons/statistics/descriptive/MinTest.java

```diff
@@ -0,0 +1,221 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.statistics.descriptive;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Arrays;
+import java.util.function.DoubleSupplier;
+import java.util.stream.DoubleStream;
+import java.util.stream.Stream;
+
+/**
+ * Test for {@link Min}.
+ */
+final class MinTest {
```

Comment:
I think we should test that the same min is returned if the input array is randomised. I would add a TestHelper class with a Fisher-Yates shuffle copied from:
[RNG ArraySampler](https://github.com/apache/commons-rng/blob/master/commons-rng-sampling/src/main/java/org/apache/commons/rng/sampling/ArraySampler.java)

If you update the signature to return the input argument then you can code:
```java
@ParameterizedTest
@MethodSource(value = "testMin")
void testMinRandomOrder(double[] values, double expected) {
    UniformRandomProvider rng = RandomSource.SPLIT_MIX_64.create();
    for (int i = 0; i < 10; i++) {
        testMin(TestHelper.shuffle(rng, values.clone()), expected);
    }
}
```

The clone is only required if the `testMin` method destructively modifies the array. Otherwise you can continue to shuffle the same array each time.

This change will require added the appropriate RNG modules to the pom.